modules/nf-core/svdb/merge/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::svdb=2.8.1
+  - bioconda::samtools=1.16.1
```

Comment:
we should update samtools here

Reply:
https://github.com/nf-core/modules/issues/4277

Reply:
Merge 5his one in?

Reply:
That's just an issue